core-modular/src/utilities/defaultLuigiTranslationTable.ts

```diff
@@ -0,0 +1,23 @@
+const defaultLuigiInternalTranslationTable: Record<string, any> = {
```

Comment:
This should be used as well. I talked with Philipp and the head lib should get the translated values.